### DISCUSSION TITLE
How to Specify Multiple Path Sources to Query template next to {Search}

### DISCUSSION THREAD

**Discussion by RamBalani** [microsoft-search/pnp-modern-search#2579] [Q&A]
I have successfully used PnP Search Results before for another application BUT there the PnP Search Results path was  pointing only to one List as Data source for the PnP Search Box, e.g. below 

`{searchTerms} path: "https://fdasmarts.sharepoint.com/sites/CDRHTownHallsCFRs2/Lists/Rev3ISO13485WithNewCol"`

How do incorporate multiple lists into the path: above?
Note:
I have set up 2 List column Managed Metadata -  1st USFDA:CFR and a 2nd 
 one is FDATopic:Biologics  or FDA Topic:Devices  List column values  in each of the Lists as appropriate  ( confirmed Managed Property mapped to crawled
 property correctly set up with successful Microsoft searches)
Stated another way- the end goal is this:
Though the Microsoft  Site search box using Managed Metadata above now works successfully with  searches - I wish to set up the same as PnP Search Result  Filters (Refinables) so **users do not need to enter: USFDA:CFR or FDATopic:Biologics** for example when searching for 'quality' NEAR 'supplier' in the PnP Search box
[Ask GitHub How to Specify Multiple Lists .docx](https://github.com/microsoft-search/pnp-modern-search/files/9878067/Ask.GitHub.How.to.Specify.Multiple.Lists.docx)

**Comment by kasperbolarsen** [comment]
Hi, are you asking two questions?

As far as the question about the two path criteria: you should be alle to use something like this {searchTerms} (path:"path1" OR path:"path2")

As far as the question about the refiners goes I assume that you are planning to use the PnP Modern Search Web parts to display the results rather than MS search.  In such case you will to map a refinablestring to the FDATopic crawled property,  wait for the mapping to take effect and add the refinablestring to selected properties in the results Web part. Finally you can add it to the refiner web part and the end users will now be able to filter the results 😁

**Comment by RamBalani** [comment]
Thanks for the tips Kasper.
But we resorted to Microsoft Search custom verticals/refinable filters and they work if an Admin or Organization member logs in.

We have many External Guest (OTP) users however and when an External Guest goes to the same site wide search page - the DEFAULT vertical/refiners show up-NOT the CUSTOM.

We reached out to Microsoft Tech support-I was advised this feature is NOT supported for External Guest -to be clear CUSTOM VERTICALS and Refiners CANNOT BE INVOKED by Site visitors though granted Site Permission if they sign up as External Guest. 
Microsoft further  stated there is no work around- HARD for me to believe since it defeats the purpose of making collaboration with Partners, external 3rd parties, etc RESTRICTED! So Microsoft is telling us only DEFAULT VERTICALS/REFINERS work for External Guests.

Turns out -after our exhaustive diagnostics on our own here is the CONCLUSION we reached-would be grand to KNOW if you AGREE OR DISAGREE with our finding and can suggest a work around.
CONCLUSION--> Only Organization Member account/users CAN access Custom Verticals/Refiners. A Microsoft account is NOT NEEDED. An Office 365 Licenses IS ALSO NOT NEEDED. But an account such as jamesbond@fdasmart.com for example WORKS and can see the Custom Verticals/Refiners we had set up. See attachment. Here's the site being accessed:  https://fdasmarts.sharepoint.com/sites/DummyCommunicationSite/
[GitHub Kasper Reply-Nov20-2022.docx](https://github.com/microsoft-search/pnp-modern-search/files/10049610/GitHub.Kasper.Reply-Nov20-2022.docx)

Ram Balani
rbalani@estarehelper.com

**Comment by wobba** [comment]
@RamBalani Microsoft (my team) is looking into supporting the same search experience for guest users as native tenant users. Will share timelines for this later as it's part of a larger initiative.

**Comment by RamBalani** [comment]
Awesome ! Many thanks Mikael for the heads-up information on this-much appreciated.

**Comment by RamBalani** [comment]
Is Github communication  censored  for content  on responses and exchanges Mikael? 
I had included other comments/feedback on this - they seem to have gone missing.
Just wondering.
Ram B

**Comment by RamBalani** [comment]
> @RamBalani Microsoft (my team) is looking into supporting the same search experience for guest users as native tenant users. Will share timelines for this later as it's part of a larger initiative.

**Comment by wobba** [comment]
@RamBalani I removed what I considerd 'ad-like' content for a solution you made/offer. And the formatting with large headlines did not help. If I misread this then I apologize. And this is not the place to discuss product opportunities with Microsoft, nor am I the right person.

**Comment by RamBalani** [comment]
I mentioned this (highlighted) clearly to make a point, i.e. our solution would not be uniquely possible were it not for Microsoft Sharepoint indexing, verticals and refinable filters with Microsoft Search.
With regards to you not being the right person for Microsoft partnering-that could have been stated without deleting my comments.

**Comment by wobba** [comment]
@RamBalani your tone does not really help if you want help. Just saying 🙂
